src/index.ts

```diff
@@ -66,12 +68,20 @@ const ifs = unixfs(helia)
 pino.logger.info(`Helia is running! PeerID: ${helia.libp2p.peerId.toString()}`)
 
 autoPeering.start()
+diskUsageCron.start()
 
 const PORT = config.serverPort
 const app = express()
 
 app.use(pino)
 
+const corsOptions = {
+  origin: config.cors.originRegexps.map((item: string) => new RegExp(`^https?:\\/\\/${item}`)),
+  methods: ['GET', 'POST']
+}
```

Comment:
What is the purpose of using RegExp?

Reply:
To support wildcards such as *.adamant.im, adm.im, *.vercel.app, *.surge.sh

Reply:
Can you specify wildcards directly in the config?